main.py

```diff
@@ -73,6 +73,9 @@ def parse_args():
     Warning: You currently cannot change/add folder structure.
     """,
     )
+    parser.add_argument("--template_idx", type=int, default=None)
```

Comment:
I'm not sure I understand the use case for a template index. If one already knows the index for the template, why not just provide the name?

Reply:
Two use cases:
- Testing: Sometimes I just want to run a task to get a feeling for the performance without caring about the template. So i just "randomly" choose the first template
- Running arrays: Again without having to lookup the template names, iterating over multiple templates by just feeding in indices

Reply:
Fair enough 👍